cuvslam-skills/cuvslam-troubleshoot/scripts/stereo_image_throttle.py

```
@@ -0,0 +1,146 @@
+import rclpy
+from rclpy.node import Node
+from rclpy.parameter import Parameter
+from sensor_msgs.msg import Image
+
+
+class StereoThrottle(Node):
+    def __init__(self) -> None:
+        super().__init__(
+            'stereo_throttle',
+            parameter_overrides=[
+                Parameter('use_sim_time', Parameter.Type.BOOL, True),
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)